README.md

```diff
@@ -1,13 +1,42 @@
 # Ledger Python API
+![](https://github.com/fetchai/ledger-api-py/workflows/CI/badge.svg)
+![](https://img.shields.io/pypi/v/fetchai-ledger-api)
+![](https://img.shields.io/github/release-date/fetchai/ledger-api-py)
+![](https://img.shields.io/pypi/pyversions/fetchai-ledger-api)
 
-Python library code for interacting with the ledger
+Python library code for interacting with the Fetch.ai ledger platform.
 
 ## Installation
 
-    $ python3 setup.py install
+    $ pip3 -U fetchai-ledger-api
+
+## Examples
+
+The project is a component of a larger ecosystem of projects for developing on the Fetch.ai's platform. Please check out some of the links below:
+
+- [Developer Portal](https://developer.fetch.ai/)
+- [Repo Examples](https://github.com/fetchai/ledger-api-py/tree/master/examples)
 
 ## Developing with the library
 
 Make sure that the library is uninstalled from the system (or use a virtualenv) then use the following commnad:
 
     $ python3 setup.py develop
```

Comment:
Is this still actually true? I'm a late comer to Python, but I've never run `setup.py` directly - it's always `pip install path/to/dir/containing/setup/file`

What does the `develop` subcommand do? We have an extra called `dev`, is that related?